libs/blocks/table-metadata/table-metadata.js

```diff
@@ -62,7 +47,6 @@ export default function init(el) {
   if (!table) return;
 
   const metadata = getMetadata(el);
-  if (metadata.highlight) handleHighlight(metadata.highlight.text, table);
```

Comment:
Do existing tables need to be re-authored to account for this change, `highlight` being removed from table metadata and moved to the `table` block directly?

Reply:
I'm not positive I understand your question.  But I think you're asking if the template of having a blank row for a highlight would still be in the template people copy from the library.  I would say you could leave the row blank, but it would make more sense to put placeholder text. If you're asking about the sample tables you may have on your test pages, yes.  If you're sample tables have a highlight row, you need to do 2 things to make them still work:
1. Move the text of the highlights from the meta data table block to the row/column of the table block where you want them
2. Add the highlight variant to the table block

If this block was already live and in use, this would be an issue.  But since the only people using it are the devs working on this block, now is the time to make this improvement.

Reply:
> If this block was already live and in use, this would be an issue. But since the only people using it are the devs working on this block, now is the time to make this improvement.

That was actually my question but then I checked that there currently is no `table` block yet, so all good. Thanks for the info 👍